Dockerfile

```diff
@@ -26,12 +36,16 @@ RUN source $HOME/.cargo/env \
 # Run Stage
 # ------------------------------------------------------------------------------
 
-FROM alpine:3.16
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7
```

Comment:
ubi9?

Reply:
I'm not familiar with all the ubi9 changes related to product builds, so I'd rather not.